packages/next-app/src/app/pool/[poolId]/MetadataAttribute.tsx

```diff
@@ -173,9 +171,7 @@ export function MetadataAttribute({ poolId }: { poolId: string }) {
         </div>
         <div className="mt-5 w-full justify-between sm:flex sm:items-center">
           <div className="flex gap-4">
-            <Button className="bg-indigo-500 text-white hover:bg-indigo-400 focus-visible:outline-indigo-500">
-              Add new attribute
-            </Button>
+            <PoolMetadataForm />
```

Comment:
`#would` make it explicit this is a modal:
```suggestion
            <PoolMetadataFormModal />
```
Also it sounds to me like a clearer (and a bit more composable) API passing the the trigger as a child component, so effectively:

```
            <PoolMetadataFormModal>
                <Button className="bg-indigo-500 text-white hover:bg-indigo-400 focus-visible:outline-indigo-500">
                  Add new attribute
                </Button>
            </PoolMetadataFormModal>
```

Reply:
That makes sense, thanks sir